File: app/src/main/java/com/gomez/herlin/logindemo/dto/BattersDto.java
package com.gomez.herlin.logindemo.dto;

import java.io.Serializable;
import java.util.List;

public class BattersDto implements Serializable {
    private List<BatterDto> batter;

    public BattersDto() {
    }

    public BattersDto(List<BatterDto> batter) {
        this.batter = batter;
    }

    public List<BatterDto> getBatter() {
        return batter;
    }

    public void setBatter(List<BatterDto> batter) {
        this.batter = batter;
    }
}

File: app/src/main/java/com/gomez/herlin/logindemo/dto/BattersDto.kt
package com.gomez.herlin.logindemo.dto

import java.io.Serializable

class BattersDto : Serializable {
    var batter: List<BatterDto>? = null

    constructor()

    constructor(batter: List<BatterDto>?) {
        this.batter = batter
    }
}
